src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.wasm.xml

```diff
@@ -0,0 +1,133 @@
+<linker>
+  <assembly fullname="*">
+    <!-- System -->
+    <type fullname="System.CLSCompliantAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ObsoleteAttribute">
```

Comment:
I suggest we add comment about things which may get broken by this - to document the fact that we understood it and are willing to live with that change of behavior.
In this case we know that XmlSerializer is using this attribute in some cases.

Reply:
Done.